spec/mailers/notifier_spec.rb

```diff
@@ -318,6 +318,26 @@
     end
   end
 
+  context "limited comments" do
+    let(:commented_limited_post) {
+      bob.post(:status_message, to: :all, public: false,
```

Comment:
This one probably happens because you aligned the argument in the next line to the call instead of indenting it.